training/datastore/utils.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+This module contains the code that deals with the PMF piece for scoring.
+
+Copyright © 2018 Red Hat Inc.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from rudra import logger
+import numpy as np
+import pandas as pd
+import json
+import io
+
+
+class Utility:
```

Comment:
Utilities don't need a class of their own IMO, because all methods would be static anyway.